Opening repositories:
C:\Users\Joshu\OneDrive\Documents\GitHub\IT1050-JoshuaWeisenburg
Opening repositories:
C:\Users\Joshu\OneDrive\Documents\GitHub\IT1050-JoshuaWeisenburg
An existing Git repository was found in C:\Users\Joshu\OneDrive\Documents\GitHub\IT1050-JoshuaWeisenburg.
An existing Git repository was found in C:\Users\Joshu\OneDrive\Documents\GitHub\IT1050-JoshuaWeisenburg.
Commit a1fd43c3 created locally in repository C:\Users\Joshu\OneDrive\Documents\GitHub\IT1050-JoshuaWeisenburg
Error: failed to push some refs to 'https://github.com/IT1050-JoshuaWeisenburg/IT1050-JoshuaWeisenburg.git'
Error: hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Error encountered while pushing to the remote repository: rejected Updates were rejected because the remote contains work that you do not have locally. This is usually caused by another repository pushing to the same ref. You may want to first integrate the remote changes before pushing again.
